UITests/UITests.App/Pages/TextBoxMask.xaml.cs

```diff
@@ -0,0 +1,62 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using System.Windows.Input;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Navigation;
+
+// The Blank Page item template is documented at https://go.microsoft.com/fwlink/?LinkId=234238
+
+namespace UITests.App.Pages
+{
+    /// <summary>
+    /// An empty page that can be used on its own or navigated to within a Frame.
+    /// </summary>
+    public sealed partial class TextBoxMask : Page, INotifyPropertyChanged
+    {
+        private const string INITIAL_VALUE = "12:50:59";
+        private const string NEW_VALUE = "00:00:00";
+
+        private string _value = INITIAL_VALUE;
+
+        public string InitialValue => INITIAL_VALUE;
+
+        public string NewValue => NEW_VALUE;
+
+        public string Value
```

Comment:
We could also use a dependency property here and not have to implement the `INotifyPropertyChanged` interface. Not sure which I'd prefer, DP may be a bit simpler for this?

Reply:
Switched to DP in 443840af69161a4baf7a7cc8f5306a32652401ca.